src/chrome/content/rules/zabmedia.ru.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="zabmedia.ru">
+	<target host="zabmedia.ru" />
+	<target host="www.zabmedia.ru" />
+	<!-- <target host="afisha.zabmedia.ru" /> --><!-- ERROR (35, schannel: SNI or certificate check failed: SEC_E_WRONG_PRINCIPAL (0x80090322) -    .) -->
+	<!-- <target host="krem.zabmedia.ru" /> --><!-- ERROR (35, schannel: SNI or certificate check failed: SEC_E_WRONG_PRINCIPAL (0x80090322) -    .) -->
```

Comment:
Please move comments on the top and also check:
miss11.zabmedia.ru
miss12.zabmedia.ru
miss13.zabmedia.ru
chudo.zabmedia.ru
www.chudo.zabmedia.ru
new.zabmedia.ru
old.zabmedia.ru
biz.zabmedia.ru
bikini.zabmedia.ru
bikini13.zabmedia.ru
persona.zabmedia.ru
turizm.zabmedia.ru
1000.zabmedia.ru
n.zabmedia.ru